src/frontend/components/PublicVideoDashboard/index.tsx

```diff
@@ -52,7 +55,12 @@ const PublicVideoDashboard = ({
         );
       case liveState.STOPPED:
       case liveState.STOPPING:
-        // nothing to show
+        // user has update permission, we redirect him to the dashboard
+        if (getDecodedJwt().permissions.can_update) {
+          return <Redirect push to={DASHBOARD_ROUTE(modelName.VIDEOS)} />;
+        }
```

Comment:
For a student, it would be better to show a message with the true situation as well? Because they most likely came here knowing that there was a live minutes ago... Something like:
> this live is now finished. If the host decides to publish the recording, the video may be available here in a while.

Reply:
Done